app/javascript/utilities/animatedImageUtils.jsx

```diff
@@ -16,13 +16,22 @@ export const initializePausableAnimatedImages = (animatedImages = []) => {
   if (animatedImages.length > 0) {
     const freezeframes = [];
 
-    // Remove the surrounding links for the image, so it can be clicked to play/pause
-    for (const image of animatedImages) {
-      image.closest('a').outerHTML = image.outerHTML;
+    for (let i = 0; i < animatedImages.length; i++) {
+      const image = animatedImages[i];
+      // Give the image an ID so that we can uniquely target it in freezeframe
+      image.setAttribute('id', `animated-${i}`);
+
+      // Our markdown parser allows use of the <center> element in posts, which could conflict with CSS we apply to center the freezeframe container
+      // Centering in this way makes sure the position is correct regardless of whether the user has used their own <center> tag
+      const center = document.createElement('center');
+      center.innerHTML = image.outerHTML;
+
+      // Remove the surrounding links for the image, so it can be clicked to play/pause
+      image.closest('a').outerHTML = center.outerHTML;
 
       freezeframes.push(
         new Freezeframe({
-          selector: `img[src="${image.getAttribute('src')}"]`,
```

Comment:
I'm not sure why the E2E tests are only surfacing this issue now, but I've switched this up to target by ID because it's possible someone would use the same gif src twice in a post 🤷‍♀️